### PR TITLE
TASK: Proper escaping for slash in full text query

### PR DESCRIPTION
**Pull request by dfeyer**
Without escaping ElasticSearch return an error for the current query.

See https://github.com/elastic/elasticsearch/issues/2980

### CODE REVIEW

**Comment by kdambekalns** [comment]
Does it really have to be `\\/`? In the ticket you linked, this is needed to avoid the shell eating the backslash, but in the single-quoted string we build in PHP…

File: Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php [hunk 1]
```diff
@@ -715,7 +715,7 @@ public function fulltext($searchWord)
     {
         $this->appendAtPath('query.filtered.query.bool.must', array(
             'query_string' => array(
-                'query' => $searchWord
+                'query' => str_replace('/', '\\/', $searchWord)
```
Comment (aertmann) [inline Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php]: hmm maybe it's actually needed to pass it through `json_encode` instead? this seems hacky at best

**Comment by dfeyer** [comment]
I will push a new version. The first version was an ugly quick fix :(

**Comment by kdambekalns** [comment]
Great, looking forward to it.

**Pull request merged**